python/tank/platform/util.py

```diff
@@ -261,3 +261,33 @@ def get_logger(module_name):
     return logging.getLogger(full_log_path)
 
 
+def get_hyperlink_html(url, name, color=None, bold=True):
+    """
+    Provides an html string for a hyperlink pointing to the given URL
+    and displaying the provided string name.
+
+    :param str url: The URL that the hyperlink navigates to.
+    :param str name: The string name to display.
+    :param str color: The color of the text. If not given, defaults to
+                      the SG_LINK_COLOR constant.
+    :param bool bold: Whether the displayed text is bold.
+    :return: HTML string.
+    """
+    from sgtk.platform.constants import SG_STYLESHEET_CONSTANTS
```

Comment:
This is no longer needed, as the utility function is now a method on the qtwidgets framework.